server.js

```diff
@@ -209,14 +212,77 @@ app.get('/leaderboard-table', function(req, res) {
 
     ft = true;
     shrt = false;
+    
+    if (req.query.fullTable == 'false')
```

Comment:
why are you adding those, are we actually ever sending fullTable  as a different value

Reply:
always use ===

Reply:
if (...) {
}
always use brackets

Reply:
Good catch again, I'll remove fullTable